portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,37 +33,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private ArrayList<String> shows;
-
+ 
   @Override
-  public void init(){
-      shows = new ArrayList<String>();
-      shows.add("The Good Doctor");
-      shows.add("Grey\'s Anatomy");
-      shows.add("House");
-      shows.add("Code Black");
-  }
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //String show = shows.get((int) (Math.random() * shows.size()));
-    //response.setContentType("text/html;");
-    //response.getWriter().println(show);
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      //shows = new ArrayList<String>();
```

Comment:
You can remove this line instead of just commenting it out.  Sometimes it is useful to leave in commented code, but only if it is only temporarily commented out and you intend to bring it back later.  Even then it is useful to add a comment describing when and why you plan on bringing it back.